src/ducks/connections/konnector.js

```diff
@@ -32,26 +34,23 @@ const reducer = (state = {}, action) => {
 export default reducer
 
 // selectors
-export const getConnectionStatus = (state) => {
-  return Object.keys(state).reduce((status, accountId) => {
-    if (hasAccountError(state[accountId])) return 'error'
-    if (isRunning(state[accountId])) return 'running'
-    if (hasRun(state[accountId])) return 'done'
-  }, 'running')
-}
-
-export const hasError = (state) => {
-  return Object.keys(state).find(accountId => hasAccountError(state[accountId]))
-}
-
-export const hasQueuedConnection = (state) => {
-  return Object.keys(state).find(accountId => isQueued(state[accountId]))
-}
-
-export const hasRunConnection = (state) => {
-  return Object.keys(state).find(accountId => hasRun(state[accountId]))
+const getKonnectorIconURL = (registryKonnector) => {
+  let icon = null
+  try {
+    icon = require(`../../assets/icons/konnectors/${registryKonnector.slug}.svg`)
```

Comment:
@ptbrowne added some changes for fetching icons here, I think we should update here too.

Reply:
@CPatchane Well spotted, I updated my PR, see this last commit https://github.com/cozy/cozy-collect/pull/275/commits/3852a8f5ba8d5094c9110e46c671960cb3082a91 (ping @ptbrowne)

Reply:
Great, thanks 👍